node/collation-generation/src/lib.rs

```diff
@@ -47,6 +48,7 @@ use std::sync::Arc;
 /// Collation Generation Subsystem
 pub struct CollationGenerationSubsystem {
 	config: Option<Arc<CollationGenerationConfig>>,
+	metrics: Metrics,
```

Comment:
as far as I see there is no way to construct `CollationGenerationSubsystem` for now?

Reply:
hm, it seems so. I guess you can add one